packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -158,10 +164,14 @@ class MessageList extends Widget<Properties> {
     const lastMessageGroup = this._messageGroups?.[this._messageGroups.length - 1];
 
     if (lastMessageGroup) {
-      const lastMessageGroupUserId = lastMessageGroup.option('items')[0].author?.id;
+      const lastMessageGroupItem = lastMessageGroup.option('items')[0];
+      const lastMessageGroupUserId = lastMessageGroupItem.author?.id;
+
+      const isTimeoutExceeded = this._isTimeoutExceeded(lastMessageGroupItem, message);
 
-      if (sender?.id === lastMessageGroupUserId) {
+      if (sender?.id === lastMessageGroupUserId && !isTimeoutExceeded) {
```

Comment:
Can we rename 'sender'? It breaks the context because it's the author of the message.

Reply:
[1d62d8ed3bd53a0a017eafe66415fe0a0ecc1f67]